internal/support/scheduler/loader.go

```diff
@@ -162,16 +162,16 @@ func loadConfigIntervals() error {
 	for i := range intervals {
 		interval := contract.Interval{
 			ID:        "",
-			Created:   0,
-			Modified:  0,
-			Origin:    0,
 			Name:      intervals[i].Name,
 			Start:     intervals[i].Start,
 			End:       intervals[i].End,
 			Frequency: intervals[i].Frequency,
 			Cron:      intervals[i].Cron,
 			RunOnce:   intervals[i].RunOnce,
 		}
+		interval.Created = 0
+		interval.Modified = 0
+		interval.Origin = 0
```

Comment:
Probably don't even need these lines since an Interval type is being instantiated and the default value of int64 is zero.

Reply:
Reverted.